client/stylesheets/_modal.scss

```diff
@@ -386,6 +386,71 @@
 
 /*~~ Button End ~~*/
 
+/*~~~~ Received Offers Main ~~~~*/
+
+.profileReceivedOffersLi:hover {
+	background-color: #d3d3d3 !important;
+}
+
+.profileReceivedOffersH2 {
+	width: 76%;
+	margin: 40px auto 30px auto !important;
+}
+.profileReceivedOffersH2 a {
+	color:#00bfff !important;
+}
+
+.profileReceivedOffersLi {
+	border-bottom: 1px solid #d3d3d3;
+	padding: 20px 0;
+	width: 75%;
+	margin: 0 auto !important;
+	font-size: 0.9rem;
+}
+
+.profileReceivedOffersLi > a {
+	font-family: 'Droid Sans', sans-serif !important;
+}
+
+.profileReceivedOffersLi a ul li {
+	display: inline-block;
+}
+
+.profileReceivedOffersInfoImg {
+	margin-left: 10px;
+}
+
+.profileReceivedOffersInfoImg img {
```

Comment:
Merge rule `.profileReceivedOffersInfoImg img` with rule on line 419<br>Selector `profileReceivedOffersInfoImg` should be written in lowercase with hyphens